lighthouse-core/gather/gatherers/dobetterweb/document-write.js

```diff
@@ -31,15 +31,14 @@ class DocWriteUse extends Gatherer {
 
   afterPass() {
     return this.collectUsage().then(DocWriteUses => {
-      this.artifact = {
+      return {
```

Comment:
I wasn't going to change the artifact shape until the next PR, so these ones are needed to put on the `usage` property